### DISCUSSION TITLE
Replace Deprecated `onAbuseLimit()` Usage

### DISCUSSION THREAD

**Discussion by korverdev** [BetaHuhn/repo-file-sync-action#186] [Ideas] [answered]
I'm getting the following warning when running this action:
[@octokit/plugin-throttling] `onAbuseLimit()` is deprecated and will be removed in a future release of `@octokit/plugin-throttling`, please use the `onSecondaryRateLimit` handler instead

**Comment by BetaHuhn** [comment, marked as answer]
Thanks for reporting this @Casey-Kiewit!

Will update the code accordingly.

**Reply by BetaHuhn** [reply]
Fixed in version [v1.17.7](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.17.7)
